### PR TITLE
Duskwhisker branch

### PR DESCRIPTION
**Pull request by dusk0220**
pronoun tagged kit thoughts + added some

### CODE REVIEW

File: resources/dicts/thoughts/alive/kitten.json [hunk 1]
```diff
@@ -26,28 +26,28 @@
     {
         "id": "gen_kit",
         "thoughts": [
-            "Plays moss ball by themselves",
-            "Wonders who their mentor will be",
+            "Plays moss ball by {PRONOUN/m_c/subject}",
```
Comment (scribblecrumb) [inline resources/dicts/thoughts/alive/kitten.json]: this should be a 'self' pronoun, not a 'subject' pronoun

File: resources/dicts/thoughts/alive/kitten.json [hunk 2]
```diff
@@ -26,28 +26,28 @@
...
+            "Wonders who {PRONOUN/m_c/subject} mentor will be",
```
Comment (scribblecrumb) [inline resources/dicts/thoughts/alive/kitten.json]: looks like a lot of the subject pronouns you've put in should be poss pronouns.  'their' is a possessive, not a subject

File: resources/dicts/thoughts/alive/kitten.json [hunk 3]
```diff
@@ -26,28 +26,28 @@
...
             "Is trying to growl menacingly",
-            "Is adamantly refusing to take their nap",
+            "Is adamantly refusing to take {PRONOUN/m_c/subject} nap",
             "Is batting pebbles across the camp clearing",
             "Is crying after rough-housing too hard with the older cats",
-            "Is regretting eating the bug that they caught",
+            "Is regretting eating the bug that {PRONOUN/m_c/subject} caught",
             "Recently took a tumble off of a log",
-            "Is busy mastering a battle move they are performing incorrectly",
+            "Is busy mastering a battle move {PRONOUN/m_c/subject} {VERB/m_c/'ve/'s} performing incorrectly",
```
Comment (scribblecrumb) [inline resources/dicts/thoughts/alive/kitten.json]: I think you need 'been' between the verb and 'performing incorrectly', else it doesn't make much sense when read?

File: resources/dicts/thoughts/alive/kitten.json [hunk 4]
```diff
@@ -59,11 +59,11 @@
         "id": "kit_older_cats",
         "thoughts": [
             "Is asking older cats how kits are born",
-            "Wishes other cats would stop babying them",
+            "Wishes other cats would stop babying {PRONOUN/m_c/subject}",
```
Comment (scribblecrumb) [inline resources/dicts/thoughts/alive/kitten.json]: you've got some 'subject' in places where they should be 'object', 'them' is an object not a subject

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

**Comment by CL0WNTH0UGHTS** [comment]
You've replaced some "they are" with /the incorrect verb tag:

It shouldn't be {VERB/m_c/'ve/'s} because it reads like

"Is thinking about how nervous they've" orr "Is thinking about how nervous she's"

It should be {VERB/m_c/are/is} so it's read as "Is thinking about how nervous they are" or "Is thinking about how nervous she is"

File: resources/dicts/thoughts/alive/kitten.json [hunk 5]
```diff
@@ -210,7 +210,7 @@
         "thoughts": [
             "Is pretending to be the medicine cat",
             "Plucks feathers from a fresh-kill bird to add to nests",
-            "Offers to leave camp to collect moss, but can't, because {PRONOUN/m_c/subject} a kit",
+            "Offers to leave camp to collect moss, but can't, because {PRONOUN/m_c/subject}{VERB/m_c/'re} a kit",
```
Comment (scribblecrumb) [inline resources/dicts/thoughts/alive/kitten.json]: missing the /'s}

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

**Review by scribblecrumb — APPROVED** [review]

**Pull request merged**